client/src/components/Ticketing/checkout/CompleteOrderForm.tsx

```diff
@@ -184,35 +184,43 @@ export default function CompleteOrderForm(
             border-zinc-300 p-4 rounded-lg " type='text' name="visit-source" onChange={(ev: React.ChangeEvent<HTMLInputElement>): void =>
                       setvisitSource(ev.target.value)} placeholder="How did you hear about us?"/>
                   </div>
-
-
-                  <input className="input w-full  border
-            border-zinc-300 p-4 rounded-lg col-span-2 " type='text' name="comments" onChange={(ev: React.ChangeEvent<HTMLInputElement>): void =>
-                    setComments(ev.target.value)} placeholder="Comments"/>
+                  <div>
+                    <span className=" block text-sm font-medium text-slate-700 ml-1">
+                    Seating Accommodations
+                    </span>
+                    <select className="input w-full  border
+            border-zinc-300 p-4 mt-1 rounded-lg col-span-2 " name="seatingAcc" value={seatingAcc} onChange={(ev: React.ChangeEvent<HTMLSelectElement>): void =>
+                      setseatingAcc(ev.target.value)}>
+                      <option value="None">No, not at this time</option>
+                      <option value="Wheel Chair">Yes, wheelchair seat(s)</option>
+                      <option value="Aisle Seat">Yes, aisle seat(s)</option>
+                      <option value="First/Ground floor">Yes, seat(s) on the ground or the first level</option>
+                      <option value="ASL Interpreter">Yes, seats in the ASL interpreters section</option>
+                      <option value="Wide Seats">Yes, wide seats</option>
+                      <option value="Other">Yes, other (describe accommodation in comment box below, if not listed)</option>
+                    </select>
+                  </div>
+                  <div>
+                    <span className=" block text-sm font-medium text-slate-700 ml-1">
+                   Comments
+                    </span>
+                    <input className="input w-full  border
+            border-zinc-300 p-4 mt-1 rounded-lg col-span-2" type='text' name="comments" onChange={(ev: React.ChangeEvent<HTMLInputElement>): void =>
+                      setComments(ev.target.value)} placeholder="Comments"/>
+                  </div>
                 </div>
-                <div className='flex flex-col items-start gap-3 mt-10'>
+                <div className='flex flex-col items-start gap-3 mt-5' mb-5>
```

Comment:
This line looks like it might not be applying the `mb-5` since its out of the quotes?

Reply:
fixed